FrontEnd/Hopsy/src/screens/HomeScreen.js

```diff
@@ -8,48 +8,11 @@ import Icon from "react-native-vector-icons/Ionicons";
 import { FlatList } from "react-native-gesture-handler";
 import moment from 'moment';
```

Comment:
for imports that will cause projects to not build without adjustments to the environment should have a warning or highlight

Reply:
Issues regarding this import have been addressed in i #175